UI/src/robot-vue.js

```diff
@@ -69,20 +69,20 @@ function interpret(machine, onChange, initialContext, event) {
 }
 
 function contextRef(service, key) {
-    let ref;
+    let _ref;
```

Comment:
why does it want to rename to _ref? Shouldn't we just be shorter ourselves and just rename it to `r`?

Reply:
Fixed when removed airbnb rules. Restored to ref.